src/css/index.css

```diff
@@ -69,18 +72,55 @@ article {
 	background-color: #fff;
 }
 
+.search {
+	text-align: center;
+	font-weight: normal;
+	margin: 1em 0;
+	@media (min-width: $breakMobileLandscape) {
+		display: flex;
+		align-items: center;
+		font-size: 1.5em;
+		* + * {
+			margin-left: 1em;
+		}
+	}
+}
+.search-input {
+	font: inherit;
+	font-weight: 300;
+	flex: 1;
+	width: 100%;
+	padding: 0 .5em;
+	line-height: 1.7em;
```

Comment:
will play with it, but mathematically pleasing numbers don't always make the most visually pleasing result.